config_manager.go

```diff
@@ -353,3 +371,12 @@ func (cm *ConfigManager) ExcludeFiles() []*regexp.Regexp {
 func (cm *ConfigManager) ExcludePatterns() []*regexp.Regexp {
 	return *cm.Config.ExcludePatterns
 }
+
+func (cm *ConfigManager) printValidationWarnings() {
+	cm.Config.printValidationWarnings()
+	for _, fileToWatch := range cm.Files() {
+		if _, err := os.Stat(fileToWatch); os.IsNotExist(err) {
+			log.Warningf("Validation: file does not exist: %s", fileToWatch)
```

Comment:
This warning duplicates [this](https://github.com/featalion/remote_syslog2/blob/fix-issue-43/remote_syslog.go#L82) error when running in the foreground. Is there a situation that the above code wouldn't catch that the linked error would?

As far as I can tell, the difference is that the linked error is checking each file continuously as r_s2 runs. However, moving a file, deleting it, re-pointing a symlink to a non-existent file, etc, may generate file notification events that we can handle separately, so checking for existence once on startup might be enough.